src/style.css

```
@@ -1,3 +1,78 @@
+/* #region aproved */
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  padding: 0;
+
+  --main-color: #00acdc;
+}
+
+.li__item {
+  list-style-type: none;
+  margin-left: 20px;
+}
+
+.li__item:first-child {
+  margin-left: 0;
+}
+
+.logo__link {
+  height: 40px;
+  width: 40px;
+}
+
+.list {
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+}
+
+/* #endregion */
+
+.header {
+  margin: 0;
+  padding: 0 50px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 2px rgba(0, 0, 0, 0.1);
+  /* background-color: #aaa; */
+}
+
+.nav {
+  height: 60px;
+  /* background-color: yellow; */
+}
+
+.a__links {
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  display: block;
+  line-height: 60px;
+  text-transform: uppercase;
+  /* background-color: aqua; */
+}
+
+.a__links.is-active {
```

Don`t increase the specification of the selector
```suggestion
.is-active {
```

---

It is very interesting, that css in your reference using this method (increasing priorities)